src/libraries/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -117,7 +119,7 @@ public IPAddress(long newAddress)
         ///     Constructor for an IPv6 Address with a specified Scope.
         ///   </para>
         /// </devdoc>
-        public IPAddress(byte[] address, long scopeid) :
+        public IPAddress(byte[]? address, long scopeid) :
```

Comment:
Since the line below is throwing on null inputs, the argument should be declared non-nullable.

Reply:
Fixed.